src/features/locators/components/LocatorListHeader.jsx

```diff
@@ -231,6 +235,13 @@ export const LocatorListHeader = ({ render }) => {
         ) : null}
       </Row>
       {render({ expandAll, setExpandAll, searchString })}
+      {isCreateModalOpen ? (
+        <LocatorEditDialog
```

Comment:
if `LocatorListHeader.jsx` would be `tsx`, you will see an error about locator related props

Reply:
fixed